service/matching/matching_engine.go

```diff
@@ -1372,34 +1378,99 @@ func (e *matchingEngineImpl) unloadTaskQueuePartition(unloadPM taskQueuePartitio
 	delete(e.partitions, key)
 	e.partitionsLock.Unlock()
 	foundTQM.Stop()
-	e.updateTaskQueueGauge(foundTQM, false, -1)
 }
 
-func (e *matchingEngineImpl) updateTaskQueueGauge(pm taskQueuePartitionManager, versioned bool, delta int) {
-	countKey := taskQueueCounterKey{
+// Responsible for emitting and updating loaded_physical_task_queue_count metric
+func (e *matchingEngineImpl) updatePhysicalTaskQueueGauge(pm *physicalTaskQueueManagerImpl, delta int) {
+
+	// calculating versioned to be one of: “unversioned” or "buildId” or “versionSet”
+	versioned := "unversioned"
+	if buildID := pm.queue.BuildId(); buildID != "" {
+		versioned = "buildId"
+	} else if versionSet := pm.queue.VersionSet(); versionSet != "" {
+		versioned = "versionSet"
+	}
+
+	PhysicalTaskQueueParameters := taskQueueCounterKey{
```

Comment:
don't use initial capital here, it's not "public"
```suggestion
	physicalTaskQueueParameters := taskQueueCounterKey{
```